features/support/response_support_methods.rb

```diff
@@ -53,6 +47,18 @@ def extract_response_section(response, version, section_name)
   relevant_section
 end
 
+# Fetch the json values from within the response based on the mapping defined for the section
+def extract_response_section(response, single_shot_response, version, section_name)
```

Comment:
Given we've got the full API response from both traditional and single-shot sources to hand, would it be worth doing a complete comparison of the two payloads, rather than just comparing the sections referenced in the test? It would be more likely to weasel out any strange side effects.

Reply:
I was worried that there would be some time-based data (assessment_id?) but I didn't actually check, I just went with this. Worth testing though, ta.